Sources/OpenAPIRuntime/Conversion/Configuration.swift

```diff
@@ -44,6 +75,11 @@ public struct ISO8601DateTranscoder: DateTranscoder {
 extension DateTranscoder where Self == ISO8601DateTranscoder {
     /// A transcoder that transcodes dates as ISO-8601–formatted string (in RFC 3339 format).
     public static var iso8601: Self { ISO8601DateTranscoder() }
+
+    /// A transcoder that transcodes dates as ISO-8601–formatted string (in RFC 3339 format) with fractional seconds.
+    public static var iso8601WithFractionalSeconds: Self {
```

Comment:
Looking for naming feedback here.

Reply:
Maybe we don't have it, but if we do I think the name is clear enough. OOI, is it a problem to include fractional seconds in `.iso8601`?

Reply:
Ah, that's sad. I'd have expected that the with fractional seconds would ensure they are present when printing, but be permissive to them not being present when parsing.

Reply:
That would be ideal, agreed.